augur/utils.py

```diff
@@ -155,13 +155,17 @@ def load_features(reference, feature_names=None):
                     if feature_names is not None: #check both tags; user may have used either
                         if "gene" in feat.qualifiers and feat.qualifiers["gene"][0] in feature_names:
                             fname = feat.qualifiers["gene"][0]
+                        elif "gene_name" in feat.qualifiers and feat.qualifiers["gene_name"][0] in feature_names:
+                            fname = feat.qualifiers["gene_name"][0]
                         elif "locus_tag" in feat.qualifiers and feat.qualifiers["locus_tag"][0] in feature_names:
                             fname = feat.qualifiers["locus_tag"][0]
                         else:
                             fname = None
                     else:
                         if "gene" in feat.qualifiers:
                             fname = feat.qualifiers["gene"][0]
+                        elif "gene_name" in feat.qualifiers:
+                            fname = feat.qualifiers["gene_name"][0]
                         else:
                             fname = feat.qualifiers["locus_tag"][0]
```

Comment:
_non-blocking_

What's the rationale for putting `gene_name` after `gene` but before `locus_tag`? Would love to have that reason in the commit message or a source comment.

Reply:
Resolved (sort of!) in 8cdca957939b68f9b64aa78bcc00fcdd7140723e. See commit message for the details.